src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -97,4 +102,43 @@ export class EditKeyboardMappingComponent implements OnInit {
   public cancel(): void {
     this.onCancel.emit()
   }
+
+  public get saveButtonErrorsTooltip(): string | undefined {
+    const invalidControlsLabels: string[] = this.findInvalidControlsLabels()
+    return invalidControlsLabels.length > 0 ? $localize`common.form-required-field.button-tooltip` + `: ${invalidControlsLabels.join(', ')}` : undefined
+  }
+
+  private findInvalidControlsLabels(): string[] {
+    const invalidControlLabels: string[] = []
+    const controls = { ...this.actionTriggerForm.controls, ...this.actionTriggerDataForm.controls }
+    for (const name in controls) {
+      if (controls[name].valid) {
+        continue
+      }
+      const translateControl: string | undefined = this.translateControl(name)
+      if (translateControl) {
+        invalidControlLabels.push(translateControl)
+      }
+    }
+    return invalidControlLabels
+  }
+
+  private translateControl(controlName: string): string | undefined {
+    switch (controlName) {
+      case FORM_TRIGGER_ON_CONTROL_LABEL:
+        return $localize`action-triggers.trigger-on.label`
+      case FORM_KEYS_CONTROL_LABEL:
+        return $localize`action-triggers.shortcut.label`
+      case FORM_ACTION_CONTROL_LABEL:
+        return $localize`action-triggers.action.label`
+      case FORM_ACTION_ARGUMENTS_CONTROL_LABEL:
+        return this.selectedAction?.argument?.name
+      default:
+        return undefined
```

Comment:
I think we should just return the untranslated control name if we can't find anything.
That way it's easier for us to spot, that we are missing an translation and we don't have to have a method that returns undefined.

Reply:
Done